react/src/components/ComputeSessionNodeItems/SessionResourceNumbers.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Currently, the detail drawer uses the `ResourceNumbersOfSession` from the `SessionLauncherPage`, did you leave this in mind for component separation to use the session node fragment later?

Reply:
Yes, let's this improve later using Fragment and file separation.